sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -367,8 +410,16 @@ case class Remainder(left: Expression, right: Expression)
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(a, b) - Returns the positive modulo",
-  extended = "> SELECT _FUNC_(10,3);\n 1")
+  usage = "_FUNC_(expr1, expr2) - Returns the positive modulo.",
+  extended = """
+    Arguments:
+      expr1 - a numeric expression.
+      expr2 - a numeric expression.
+
+    Examples:
+      > SELECT _FUNC_(10, 3);
```

Comment:
I think this definitely needs an example with a negative argument too, to show for example that PMOD(-10,3) is 2 but -10 % 3 is -1.